promscale/installation/promscale-with-timescale-cloud/docker.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Install Promscale using a Docker image
+excerpt: Install Promscale using Docker
+product: promscale
+keywords: [analytics, Docker]
+tags: [install]
+related_pages:
+  - /promscale/:currentVersion:/guides/resource-recomm/
+  - /promscale/:currentVersion:/send-data/
+---
+
+import PromscaleSendData from "versionContent/_partials/_promscale-send-data.mdx";
+
+# Install Promscale Connector using a Docker image with Timescale Cloud
+
+<highlight type="important">
+Running Promscale directly using `docker run` is not recommended for production
+environments. This can be useful for testing purposes and is just provided as an
+example.
```

Comment:
```suggestion
Running Promscale from a Docker container is not suitable for production
environments. This can be useful for testing purposes and is provided here as an
example only.
```